### PR TITLE
MSBot: Clean up orphaned bot file on cancellation of clone service, added doc for special characters in appSecret

### PR DESCRIPTION
**Pull request by vishwacsena**
Fixes 
- #649 
- #579 
- #590

### CODE REVIEW

File: packages/MSBot/docs/export-clone.md [hunk 1]
```diff
@@ -57,3 +57,12 @@ msbot clone services --folder folderName --name NewBot --luisAuthoringKey 000000
 | -q, --quiet                                 | disable output                                                                                                       |
 | --prefix                                    | Append [msbot] prefix to all messages                                                                                |
 | -h, --help                                  | output usage information                                                                                             |
+
+
+> **NOTE:** If you are explicitly passing in appSecret as an argument you need to escape any special characters in the password that might be interpreted by the shell to be a command.
```
Comment (carlosscastro) [inline packages/MSBot/docs/export-clone.md]: Thanks for this!

File: packages/MSBot/src/msbot-clone-services.ts [hunk 2]
```diff
@@ -240,6 +245,8 @@ async function processConfiguration(): Promise<void> {
             if (!args.force) {
```
Comment (carlosscastro) [inline packages/MSBot/src/msbot-clone-services.ts]: If force true, do we still remove the orphaned bot file?

Comment (carlosscastro) [reply packages/MSBot/src/msbot-clone-services.ts]: Or should we remove it?

Comment (vishwacsena) [reply packages/MSBot/src/msbot-clone-services.ts]: If user specified the force flat, then we will just go ahead with cloning without prompting user to confirm. There is no need to remove the bot file in that case.

**Pull request merged**